File: dmp-evaluator/src/main/java/org/arnhold/evaluation/EvaluationApplication.java
package org.arnhold.evaluation;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.boot.autoconfigure.jdbc.DataSourceAutoConfiguration;
import org.springframework.context.annotation.ComponentScan;

@SpringBootApplication(exclude = {DataSourceAutoConfiguration.class })
@ComponentScan(basePackages = {"at.ac.tuwien", "org.arnhold"})
public class EvaluationApplication {
	public static void main(String[] args) {
		SpringApplication.run(EvaluationApplication.class, args);
	}

}

File: evaluator/src/main/kotlin/org/arnhold/evaluation/EvaluationApplication.kt
package org.arnhold.evaluation

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.context.annotation.ComponentScan

@SpringBootApplication
@ComponentScan(basePackages = ["org.arnhold"])
class EvaluationApplication

fun main(args: Array<String>) {
    runApplication<EvaluationApplication>(*args)
}

